dev/setup.sh

```
@@ -75,6 +75,7 @@ kubectl label ns/$FOURNOS_NAMESPACE fournos.dev/queue-access=true
 echo ""
 echo "Applying FournosJob CRD..."
 kubectl apply -f manifests/crd.yaml -n $FOURNOS_NAMESPACE
+kubectl apply -f manifests/crd-jobconfig.yaml -n $FOURNOS_NAMESPACE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)